File: commands/beajpeg.py
import bot
import discord
from random import choice


def import_command():
    @bot.tree.command(
        name="beajpeg",
        description="Send a Random Image of Frank"
    )
    async def be_a_jpeg(interaction: discord.Interaction):
        await interaction.response.send_message(file=discord.File(choice(bot.assets)))

File: src/commands/fun/beajpeg.js
const fs = require("fs");
const path = require("path");
const { assets } = require("../../index.js");
const { SlashCommandBuilder } = require("discord.js");

module.exports = {
  data: new SlashCommandBuilder()
    .setName("beajpeg")
    .setDescription("Send a Random Image of Frank"),
  async execute(ctx) {
    let picture = path.join(
      __dirname,
      "../../../assets",
      assets[Math.floor(Math.random() * assets.length)]
    );
    await ctx.reply({ files: [{ attachment: picture }] });
  },
};
